ruff_lsp/server.py

```diff
@@ -1167,6 +1170,15 @@ async def _run_check_on_document(
         else:
             argv.append(arg)
 
+    # If the Ruff version is sufficiently recent, use the deprecated `--format`
+    # argument instead of `--output-format`.
+    if not VERSION_REQUIREMENT_OUTPUT_FORMAT.contains(
+        executable.version, prereleases=True
+    ):
+        index = argv.index("--output-format")
+        argv.pop(index)
+        argv.insert(index, "--format=json")
```

Comment:
Should the second arg here be `--format`, since `json` is provided as a separate entry?

Reply:
Yeah thanks — copilot inserted that -.-